lib_xud/api/xud.h

```diff
@@ -20,6 +20,34 @@
 #include "xud_conf.h"
 #endif
 
+#ifndef XUD_USB_ISO_EP_MAX_TXN_SIZE
```

Comment:
I think this needs adding to options.rst

Reply:
done